Include some headers file, to fix a problem with getrusage and setpriority functions with the new ubuntu raring ringtail development release
Author: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
Forwarded: <yes>
Fixed in upstream revision 9223fd2
http://boinc.berkeley.edu/trac/changeset/9223fd22e3b6b848cc9e5e151d958542da8fd49d/boinc


--- boinc-7.0.36+dfsg.orig/samples/wrapper/wrapper.cpp
+++ boinc-7.0.36+dfsg/samples/wrapper/wrapper.cpp
@@ -31,6 +31,8 @@
 #include <stdio.h>
 #include <vector>
 #include <string>
+#include <sys/time.h>
+#include <sys/resource.h> 
 #ifdef _WIN32
 #include "boinc_win.h"
 #include "win_util.h"
